source/user-manual/capabilities/vulnerability-detection.rst

```diff
@@ -35,16 +36,28 @@ The following table shows the operating systems that the vulnerability detector
 +---------------+-------------+---------------------+
 | Distribution  | Versions    | Configuration OVALs |
 +===============+=============+=====================+
-| Redhat &      | 6           | Redhat 6 OVAL       |
-+ CentOS        +-------------+---------------------+
+|               | 5           | Redhat 5 OVAL       |
++ Redhat &      +-------------+---------------------+
```

Comment:
Please modify "Redhat" by "Red Hat" or "RedHat", we have different names along this guide.